app/src/main/java/com/robotemi/welcomingbtob/tips/TipsViewImpl.kt

```diff
@@ -57,15 +57,6 @@ class TipsViewImpl : RelativeLayout {
         val tips = mutableListOf<String>()
         when (robot.wakeupWord.toLowerCase(Locale.ENGLISH)) {
             Constants.WAKEUP_WORD_ALEXA,
-            Constants.WAKEUP_WORD_ALEXA_JAPAN -> {
-                tips.clear()
-                tips.add(String.format("Try “Alexa, tell my temi to go to %s”", locationForTips))
-                tips.add("Try “Alexa, tell my temi to follow me”")
-                tips.add(String.format("Try “Alexa, tell my temi to call %s”", callForTips))
-                tips.add("Try “Alexa, tell my temi to take a selfie”")
-                tips.add("Try “Alexa, tell my temi to take a video”")
-                tips.add("Try “Alexa, tell my temi to take a GIF”")
```

Comment:
Why delet all the tips related Alexa even this is not only for Alexa Japan?

Reply:
You're right. I meant to remove it only for Japan. I'll make the appropriate changes.